graph/src/util/futures.rs

```diff
@@ -28,13 +51,15 @@ where
         let delay_ms = 1 << attempt_count;
 
         try_it().map_err(move |e| {
-            warn!(
-                logger,
-                "Trying again in ~{} seconds after {} failed (attempt #{})",
-                (delay_ms as f64 / 1000.0).round(),
-                &operation_name,
-                attempt_count + 1,
-            );
+            if attempt_count >= log_after {
+                warn!(
```

Comment:
Can we make this a `debug!` instead of `warn!`? See my earlier comment on the PR.

Reply:
I like it! Done.